src/pages/patientView/PatientViewPageUtils.tsx

```diff
@@ -0,0 +1,19 @@
+import SampleManager from "./SampleManager";
```

Comment:
seems like these first two imports are not used

Reply:
Unused imports removed.